### ISSUE TITLE
Crash while loading a village

### ISSUE THREAD

**Issue by Carsondd** [AllTheMods/ATM-8#109]
Here is the crash report

```
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  EXCEPTION_ACCESS_VIOLATION (0xc0000005) at pc=0x00007ff824963950, pid=13388, tid=16004
#
# JRE version: OpenJDK Runtime Environment Microsoft-32931 (17.0.3+7) (build 17.0.3+7-LTS)
# Java VM: OpenJDK 64-Bit Server VM Microsoft-32931 (17.0.3+7-LTS, mixed mode, tiered, compressed oops, compressed class ptrs, g1 gc, windows-amd64)
# Problematic frame:
# C  [nvoglv64.dll+0x1433950]
#
# No core dump will be written. Minidumps are not enabled by default on client versions of Windows
#
# If you would like to submit a bug report, please visit:
#   https://github.com/microsoft/openjdk/issues
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#

---------------  S U M M A R Y ------------

Command Line: -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Dos.name=Windows 10 -Dos.version=10.0 -Xss1M -Djava.library.path=. -Dminecraft.launcher.brand=minecraft-launcher -Dminecraft.launcher.version=2.2.12145 -Djava.net.preferIPv6Addresses=system -DignoreList=bootstraplauncher,securejarhandler,asm-commons,asm-util,asm-analysis,asm-tree,asm,JarJarFileSystems,client-extra,fmlcore,javafmllanguage,lowcodelanguage,mclanguage,forge-,forge-43.1.38.jar,forge-43.1.38 -DmergeModules=jna-5.10.0.jar,jna-platform-5.10.0.jar -DlibraryDirectory=D:\Minecraft\Install\libraries --module-path=D:\Minecraft\Install\libraries/cpw/mods/bootstraplauncher/1.1.2/bootstraplauncher-1.1.2.jar;D:\Minecraft\Install\libraries/cpw/mods/securejarhandler/2.1.4/securejarhandler-2.1.4.jar;D:\Minecraft\Install\libraries/org/ow2/asm/asm-commons/9.3/asm-commons-9.3.jar;D:\Minecraft\Install\libraries/org/ow2/asm/asm-util/9.3/asm-util-9.3.jar;D:\Minecraft\Install\libraries/org/ow2/asm/asm-analysis/9.3/asm-analysis-9.3.jar;D:\Minecraft\Install\libraries/org/ow2/asm/asm-tree/9.3/asm-tree-9.3.jar;D:\Minecraft\Install\libraries/org/ow2/asm/asm/9.3/asm-9.3.jar;D:\Minecraft\Install\libraries/net/minecraftforge/JarJarFileSystems/0.3.16/JarJarFileSystems-0.3.16.jar --add-modules=ALL-MODULE-PATH --add-opens=java.base/java.util.jar=cpw.mods.securejarhandler --add-opens=java.base/java.lang.invoke=cpw.mods.securejarhandler --add-exports=java.base/sun.security.util=cpw.mods.securejarhandler --add-exports=jdk.naming.dns/com.sun.jndi.dns=java.naming -Xmx6144m -Xms256m -Dminecraft.applet.TargetDirectory=D:\Minecraft\Instances\All the Mods 8 - ATM8 - 1.19.2 -Dfml.ignorePatchDiscrepancies=true -Dfml.ignoreInvalidMinecraftCertificates=true -Duser.language=en -Duser.country=US -Dlog4j.configurationFile=D:\Minecraft\Install\assets\log_configs\client-1.12.xml cpw.mods.bootstraplauncher.BootstrapLauncher --username Billyeatspies --version forge-43.1.38 --gameDir D:\Minecraft\Instances\All the Mods 8 - ATM8 - 1.19.2 --assetsDir D:\Minecraft\Install\assets --assetIndex 1.19 --uuid 91238e485a1b4c1d8bebe72bbf589ec8 --clientId OGY0YWNmNWEtOWRmMC00ZTQ2LWEwMzYtOTUxZmU2N2MwZTgz --xuid 2535446184709953 --userType msa --versionType release --width 1024 --height 768 --launchTarget forgeclient --fml.forgeVersion 43.1.38 --fml.mcVersion 1.19.2 --fml.forgeGroup net.minecraftforge --fml.mcpVersion 20220805.130853

Host: Intel(R) Core(TM) i7-7700 CPU @ 3.60GHz, 8 cores, 15G,  Windows 10 , 64 bit Build 19041 (10.0.19041.1889)
Time: Fri Oct 14 18:33:47 2022 Eastern Daylight Time elapsed time: 228.955008 seconds (0d 0h 3m 48s)

---------------  T H R E A D  ---------------

Current thread (0x000001867c0f5540):  JavaThread "Render thread" [_thread_in_native, id=16004, stack(0x00000031b7800000,0x00000031b7900000)]

Stack: [0x00000031b7800000,0x00000031b7900000],  sp=0x00000031b78fd088,  free space=1012k
Native frames: (J=compiled Java code, j=interpreted, Vv=VM code, C=native code)
C  [nvoglv64.dll+0x1433950]

Java frames: (J=compiled Java code, j=interpreted, Vv=VM code)
J 21654  org.lwjgl.opengl.GL11C.nglDrawElements(IIIJ)V org.lwjgl.opengl@3.3.1+7 (0 bytes) @ 0x0000018608909ef6 [0x0000018608909ea0+0x0000000000000056]
J 29063 c2 com.mojang.blaze3d.vertex.VertexBuffer.m_166882_()V minecraft@1.19.2 (22 bytes) @ 0x0000018607fb6024 [0x0000018607fb5fa0+0x0000000000000084]
j  com.mojang.blaze3d.vertex.VertexBuffer.m_166876_(Lcom/mojang/math/Matrix4f;Lcom/mojang/math/Matrix4f;Lnet/minecraft/client/renderer/ShaderInstance;)V+288 minecraft@1.19.2
j  com.mojang.blaze3d.vertex.VertexBuffer.m_166867_(Lcom/mojang/math/Matrix4f;Lcom/mojang/math/Matrix4f;Lnet/minecraft/client/renderer/ShaderInstance;)V+25 minecraft@1.19.2
j  dan200.computercraft.client.render.TileEntityMonitorRenderer.renderTerminal(Lcom/mojang/math/Matrix4f;Ldan200/computercraft/shared/peripheral/monitor/ClientMonitor;FF)V+386 computercraft@1.100.10
j  dan200.computercraft.client.render.TileEntityMonitorRenderer.render(Ldan200/computercraft/shared/peripheral/monitor/TileMonitor;FLcom/mojang/blaze3d/vertex/PoseStack;Lnet/minecraft/client/renderer/MultiBufferSource;II)V+342 computercraft@1.100.10
j  dan200.computercraft.client.render.TileEntityMonitorRenderer.m_6922_(Lnet/minecraft/world/level/block/entity/BlockEntity;FLcom/mojang/blaze3d/vertex/PoseStack;Lnet/minecraft/client/renderer/MultiBufferSource;II)V+13 computercraft@1.100.10
j  net.minecraft.client.renderer.blockentity.BlockEntityRenderDispatcher.m_112284_(Lnet/minecraft/client/renderer/blockentity/BlockEntityRenderer;Lnet/minecraft/world/level/block/entity/BlockEntity;FLcom/mojang/blaze3d/vertex/PoseStack;Lnet/minecraft/client/renderer/MultiBufferSource;)V+59 minecraft@1.19.2
j  net.minecraft.client.renderer.blockentity.BlockEntityRenderDispatcher.m_112297_(Lnet/minecraft/client/renderer/blockentity/BlockEntityRenderer;Lnet/minecraft/world/level/block/entity/BlockEntity;FLcom/mojang/blaze3d/vertex/PoseStack;Lnet/minecraft/client/renderer/MultiBufferSource;)V+6 minecraft@1.19.2
j  net.minecraft.client.renderer.blockentity.BlockEntityRenderDispatcher$$Lambda$50318+0x00000008057625d0.run()V+20 minecraft@1.19.2
j  net.minecraft.client.renderer.blockentity.BlockEntityRenderDispatcher.m_112278_(Lnet/minecraft/world/level/block/entity/BlockEntity;Ljava/lang/Runnable;)V+1 minecraft@1.19.2
j  net.minecraft.client.renderer.blockentity.BlockEntityRenderDispatcher.m_112267_(Lnet/minecraft/world/level/block/entity/BlockEntity;FLcom/mojang/blaze3d/vertex/PoseStack;Lnet/minecraft/client/renderer/MultiBufferSource;)V+118 minecraft@1.19.2
j  me.jellysquid.mods.sodium.client.render.SodiumWorldRenderer.renderTileEntities(Lcom/mojang/blaze3d/vertex/PoseStack;Lnet/minecraft/client/renderer/RenderBuffers;Lit/unimi/dsi/fastutil/longs/Long2ObjectMap;Lnet/minecraft/client/Camera;F)V+240 rubidium@0.6.2
j  net.minecraft.client.renderer.LevelRenderer.handler$zbo000$onRenderTileEntities(Lcom/mojang/blaze3d/vertex/PoseStack;FJZLnet/minecraft/client/Camera;Lnet/minecraft/client/renderer/GameRenderer;Lnet/minecraft/client/renderer/LightTexture;Lcom/mojang/math/Matrix4f;Lorg/spongepowered/asm/mixin/injection/callback/CallbackInfo;)V+16 minecraft@1.19.2
j  net.minecraft.client.renderer.LevelRenderer.m_109599_(Lcom/mojang/blaze3d/vertex/PoseStack;FJZLnet/minecraft/client/Camera;Lnet/minecraft/client/renderer/GameRenderer;Lnet/minecraft/client/renderer/LightTexture;Lcom/mojang/math/Matrix4f;)V+2116 minecraft@1.19.2
j  net.minecraft.client.renderer.GameRenderer.m_109089_(FJLcom/mojang/blaze3d/vertex/PoseStack;)V+657 minecraft@1.19.2
J 23367 c1 net.minecraft.client.renderer.GameRenderer.m_109093_(FJZ)V minecraft@1.19.2 (885 bytes) @ 0x00000186023cc76c [0x00000186023cb700+0x000000000000106c]
J 46771 c1 net.minecraft.client.Minecraft.m_91383_(Z)V minecraft@1.19.2 (1221 bytes) @ 0x00000186031a3acc [0x00000186031a0c20+0x0000000000002eac]
j  net.minecraft.client.Minecraft.m_91374_()V+138 minecraft@1.19.2
j  net.minecraft.client.main.Main.m_239872_([Ljava/lang/String;Z)V+1488 minecraft@1.19.2
j  net.minecraft.client.main.Main.main([Ljava/lang/String;)V+25 minecraft@1.19.2
v  ~StubRoutines::call_stub
j  jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Ljava/lang/reflect/Method;Ljava/lang/Object;[Ljava/lang/Object;)Ljava/lang/Object;+0 java.base@17.0.3
j  jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Ljava/lang/Object;[Ljava/lang/Object;)Ljava/lang/Object;+133 java.base@17.0.3
j  jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Ljava/lang/Object;[Ljava/lang/Object;)Ljava/lang/Object;+6 java.base@17.0.3
j  java.lang.reflect.Method.invoke(Ljava/lang/Object;[Ljava/lang/Object;)Ljava/lang/Object;+59 java.base@17.0.3
j  net.minecraftforge.fml.loading.targets.CommonClientLaunchHandler.lambda$launchService$0(Ljava/lang/ModuleLayer;[Ljava/lang/String;)V+40 fmlloader@1.19.2-43.1.38
j  net.minecraftforge.fml.loading.targets.CommonClientLaunchHandler$$Lambda$890+0x00000008004b5420.run()V+8 fmlloader@1.19.2-43.1.38
j  cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch([Ljava/lang/String;Ljava/lang/ModuleLayer;)V+11 cpw.mods.modlauncher@10.0.8
j  cpw.mods.modlauncher.LaunchServiceHandler.launch(Ljava/lang/String;[Ljava/lang/String;Ljava/lang/ModuleLayer;Lcpw/mods/modlauncher/TransformingClassLoader;Lcpw/mods/modlauncher/LaunchPluginHandler;)V+58 cpw.mods.modlauncher@10.0.8
j  cpw.mods.modlauncher.LaunchServiceHandler.launch(Lcpw/mods/modlauncher/ArgumentHandler;Ljava/lang/ModuleLayer;Lcpw/mods/modlauncher/TransformingClassLoader;Lcpw/mods/modlauncher/LaunchPluginHandler;)V+21 cpw.mods.modlauncher@10.0.8
j  cpw.mods.modlauncher.Launcher.run([Ljava/lang/String;)V+310 cpw.mods.modlauncher@10.0.8
j  cpw.mods.modlauncher.Launcher.main([Ljava/lang/String;)V+78 cpw.mods.modlauncher@10.0.8
j  cpw.mods.modlauncher.BootstrapLaunchConsumer.accept([Ljava/lang/String;)V+1 cpw.mods.modlauncher@10.0.8
j  cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(Ljava/lang/Object;)V+5 cpw.mods.modlauncher@10.0.8
j  cpw.mods.bootstraplauncher.BootstrapLauncher.main([Ljava/lang/String;)V+515 cpw.mods.bootstraplauncher@1.1.2
v  ~StubRoutines::call_stub

siginfo: EXCEPTION_ACCESS_VIOLATION (0xc0000005), reading address 0x0000000000000000


```

**Comment by ddevoykirito** [comment]
I have had the same thing happen to me

**Comment by LucoEldritch** [comment]
**tl;dr: It _seems_ to be related to CC: Tweaked**

I'd like to add some information:
I'm hosting a server, where I can enter villages normally, but one of my friends is getting this crash. But, I was ALSO having visual glitches when entering villages, and I noticed that this crash report is on "Render Thread", so I suspect those problems are related.

I also noticed that there were some houses with ComputerCraft's items (CC: Tweaked more specifically), and that they were with a side missing (you can see thru), so I removed all of those computers form the villages and my visual glitches stopped. Then, I called my friend in, and his crash also stopped.

**Comment by Carsondd** [comment]
Yeah reading the error report I was thinking it was cc related. Unfortunately I cant remove the mod from the pack. Nor can I find away to disable that block type.

**Comment by QDaniel** [comment]
Same Error

**Comment by ddevoykirito** [comment]
Is there a possibility of this being fixed in the foreseeable future, and if so do you have an ETA on when that will be fixed?

**Comment by Jonz00r** [comment]
#120 someone posted a possible workaround

**Comment by whatthedrunk** [comment]
Added the config change in the newest update. When CC updates we will remove the client config.

**Comment by Meril502** [comment]
> Added the config change in the newest update. When CC updates we will remove the client config.

This problem has come back on the newest version of ATM8 1.0.11.
